lib/src/base-styles/_nx-radio-checkbox.scss

```diff
@@ -20,6 +20,18 @@
   // the end of the text
   width: calc(min(#{$max-width}, 100%));
   max-width: max-content;
+  position: relative;
+
+  &::after {
+    border: 1px solid var(--nx-color-interactive-border-focus);
+    box-shadow: var(--nx-box-shadow-focus);
+    content: '';
+    display: none;
+    height: 22px;
```

Comment:
I worry a little bit about the fact that the width and height here are assuming the dimesions of the control, even though those dimensions are defined in the individual `NxCheckbox.scss` and `NxRadio.scss` files rather than here.  it might make sense to move the `height`, `width`, and maybe `left` to those files

Reply:
> I worry a little bit about the fact that the width and height here are assuming the dimesions of the control, even though those dimensions are defined in the individual `NxCheckbox.scss` and `NxRadio.scss` files rather than here. It might make sense to move the `height`, `width`, and maybe `left` to those files

Separating them out makes sense. I initially had them grouped because the `width`, `height` and `left` were the same for both the radio and the checkboxe. I have implemented the change.
aa7d9db5def38dd8cfb54ae2af7e45f49d3f4fba

Reply:
> I initially had them grouped because the width, height and left were the same for both the radio and the checkboxe.

Right you just have to consider whether they're the same due to some fundamental link or just coincidentally.  Since the dimensions of the checkbox box and radio circle are defined in their respective files and not the shared file, that would suggest that they're only coincidentally the same.